src/metabase/automagic_dashboards/populate.clj

```diff
@@ -0,0 +1,209 @@
+(ns metabase.automagic-dashboards.populate
+  "Create and save models that make up automagic dashboards."
+  (:require [clojure.string :as str]
+            [clojure.tools.logging :as log]
+            [metabase.api.common :as api]
+            [metabase.automagic-dashboards.filters :as magic.filters]
+            [metabase.models.card :as card]
+            [toucan.db :as db]))
+
+(def ^Long grid-width
+  "Total grid width."
+  18)
+(def ^Long default-card-width
+  "Default card width."
+  6)
+(def ^Long default-card-height
+  "Default card height"
+  4)
+
+(defn- create-collection!
+  [title color description]
+  (when api/*is-superuser?*
```

Comment:
What's the intended behavior of non superusers here? Does everything have to deal with possible nil values from this?

Reply:
I'm not sure I'd characterize users spewing cards into the everything questions space  as working ok ;)